fastkml/kml.py

```diff
@@ -982,7 +982,7 @@ def shape(self, value):
         elif value is None:
             self._shape = None
         else:
-            raise ValueError
+            raise ValueError("Shape must be one of " "rectangle, cylinder, sphere")
```

Comment:
on line 1172 (GroundOverlay altitudeMode setter), the existing code sets to default value on invalid entries. Should I follow this or just raise an exception (which I think makes more sense)?

Reply:
With type annotations, this check could be dropped entirely